firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -163,6 +165,14 @@ public void setUp() throws FirebaseInstallationServiceException {
     when(persistedFidReturnsError.readPersistedFidEntryValue()).thenReturn(null);
     when(mockUtils.createRandomFid()).thenReturn(TEST_FID_1);
     when(mockClock.currentTimeMillis()).thenReturn(TEST_CREATION_TIMESTAMP_1);
+    doNothing()
+        .when(backendClientReturnsOk)
+        .deleteFirebaseInstallation(anyString(), anyString(), anyString(), anyString());
+    doThrow(
+            new FirebaseInstallationServiceException(
+                "Server Error", FirebaseInstallationServiceException.Status.SERVER_ERROR))
+        .when(backendClientReturnsError)
+        .deleteFirebaseInstallation(anyString(), anyString(), anyString(), anyString());
```

Comment:
this would benefit from a comment and maybe combining the doNothing and doThrow calls to make it clear that you're setting up subsequent calls

Reply:
Added comments. They are not subsequent calls. They use different mocks- backendClientReturnsOk & backendClientReturnsError

Reply:
ok, got it